azure/Guidelines.md

```diff
@@ -18,6 +20,35 @@ Teams building ARM Resource Providers (RPs) MUST follow the additional guidance
 
 ARM RPs are Azure Fundamentals requirement for Azure Services and ARM RP review is another mandatory review. Some of the guidance overlaps with general API review, but passing one review will generally make the other one go very quickly.
 
+### Advice for new services
+
+Developing a new service requires the development of at least 1 (management plane) API and potentially one or more additional (data plane) APIs.  
+
+* Think about naming from the context of a **developer experience**.  
+  * Start with the "things" your API manipulates, then think about the operations that a developer needs to do to these "things".
```

Comment:
@johanste should we have some guideline around the use of verbs/nouns in REST APIs, similar to Powershell best practices?

Reply:
Try to keep verbs in the present (not past or future)